runconfig/parse_test.go

```diff
@@ -322,10 +322,10 @@ func TestParseDevice(t *testing.T) {
 			PathInContainer:   "/something",
 			CgroupPermissions: "rwm",
 		},
-		"/dev/snd:/something:ro": {
+		"/dev/snd:/something:rw": {
 			PathOnHost:        "/dev/snd",
 			PathInContainer:   "/something",
-			CgroupPermissions: "ro",
+			CgroupPermissions: "rw",
 		},
```

Comment:
Ah right, it's probably better in `TestValidateDevice` :wink:. You could add a few more invalid cases (the one @calavera gave), but other than that :+1:.